### PR TITLE
docs: single run does not exist anymore

### PR DESCRIPTION
**Pull request by AnObstacle**
and fix a typo (the flag is browsers and not browser)

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
The command `ng test --single-run ...` is not working with angular 7.

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by mjawulski** [comment]
I have opened similar PR but changed wrong flags everywhere not only in one place. You can check details here: #27334

**Review by IgorMinar — CHANGES_REQUESTED** [review]

File: aio/content/guide/testing.md [hunk 1]
```diff
@@ -218,7 +218,7 @@ exports.config = config;
 Now you can run the following commands to use the `--no-sandbox` flag:
 
 <code-example language="sh" class="code-shell">
-  ng test --single-run --no-progress --browser=ChromeHeadlessCI
+  ng test --watch=false --no-progress --browsers=ChromeHeadlessCI
```
[the remark this answers is not in the document]
Comment (Mathou54) [reply aio/content/guide/testing.md]: It has changed with version 7 (I think) of the CLI. Now the default for the watch flag is true.

https://github.com/angular/angular-cli/pull/13051/files

**Comment by brandonroberts** [comment]
@Mathou54 will you rebase on master to see if these changes are still needed?

**Comment by Mathou54** [comment]
It has been changed with `--no-watch` and this flag is working.

**Pull request closed without merging**